pandas/_libs/groupby.pyx

```diff
@@ -1197,6 +1199,12 @@ cdef group_min_max(groupby_t[:, ::1] out,
         True if `values` contains datetime-like entries.
     compute_max : bint, default True
         True to compute group-wise max, False to compute min
+    mask_in : ndarray[bool, ndim=2], optional
```

Comment:
Since the `group_max` that wraps this uses `mask`, use `mask` here as well?

Reply:
good catch, updated